doc/dev/cephfs-mirroring.rst

```diff
@@ -505,7 +505,7 @@ Re-adding Peers
 When re-adding (reassigning) a peer to a file system in another cluster, ensure
 that all mirror daemons have stopped synchronizing with the peer. This can be
 checked via the  ``fs mirror status`` admin socket command (the ``Peer UUID``
-should not show up in the command output). We recommend purging 
+should not show up in the command output). We recommend purging
```

Comment:
Suggest making the parenthetical a separate sentence (we see this a lot in the docs and IMHO it's cleaner this way).

See what I did there?

Reply:
This doc specifically has tons of it and it was the second thing that stood out.

The fist thing that stood out was that possibly this document shouldn't exist at all. Did you see my comment about https://docs.ceph.com/en/latest/cephfs/cephfs-mirroring/ which is almost a copy of this? Before we get deeper into improvements....

Reply:
I have about 300 new messages to work through since yesterday, haven't seen that yet. 

Suggest we fix this here for now and put in a tracker to consider merging the two pages.

Reply:
@bluikko, I'll review both of these documents and send you a proposal to merge them, if it seems like a good idea.